docs/docs/features/mobile-app.mdx

```diff
@@ -28,6 +31,22 @@ The beta release channel allows users to test upcoming changes before they are o
 You can enable automatic backup on supported devices. For more information see [Automatic Backup](/docs/features/automatic-backup.md).
 :::
 
+## Sync only selected photos
+If you have large number of photos on the device, and you would prefer not to backup all the photos, then it might be prudent to only backup selected photos from device to the Immich server.
```

Comment:
```suggestion
If you have a large number of photos on the device, and you would prefer not to backup all the photos, then it might be prudent to only backup selected photos from device to the Immich server.
```